src/components/label/label.tsx

```diff
@@ -83,12 +87,12 @@ export class Label {
   //--------------------------------------------------------------------------
 
   connectedCallback(): void {
-    this.calciteInternalLabelConnected.emit();
+    this.calciteInternalLabelRegister.emit();
   }
 
   disconnectedCallback(): void {
     // Dispatching to body in order to be listened by other elements that are still connected to the DOM.
-    document.body?.dispatchEvent(new CustomEvent("calciteInternalLabelUnregister"));
+    document?.dispatchEvent(new CustomEvent(labelDisconnectedEvent));
```

Comment:
nitpick> I don't think the `?` is needed for document. We can remove the question mark here and any other uses.